### ISSUE TITLE
Does embedding support multi languages? 

### ISSUE THREAD

**Issue by simjak** [tjmlabs/ColiVara#126]
Hey,

I'm looking for a multi-lingual solution for financial documents.
Does the embedding document support multiple languages?

How is the chunking done in Colivara? Could you point me to the docs?

**Comment by Jonathan-Adly** [comment]
Yes - it is multi-lingual and it supports multiple language. We the use the Vidore benchmark for our evals, so the evals in the readme all for multi languages. 

The chunking is done on page-level and here are the docs: https://docs.colivara.com/

Bear in mind - ColPali and by extension ColiVara uses multi-vector (each chunk/page gets multiple vectors, instead of just one) and late interaction style calculation. Most vector DBs do not support that yet - so, we recommend using the upsert endpoint with search. Instead of trying to store the embeddings yourself.
